_dasldemos/fh-cbio-intro.md

```diff
@@ -0,0 +1,205 @@
+---
+title: cBioPortal - the Fred Hutch instance
+last_modified_at: 2024-09-13
+main_authors: sitapriyamoorthi, tefirman, laderast
+primary_reviewers: laderast, sonu, sckott
+---
+Learning Objectives
+-------------------
+
+After reading this article, you will learn:
+
+-   what **cBioPortal** is and how it can help with your research.
+-   what the **Fred Hutch instance of cBioPortal** is and steps to take to upload your data onto the instance.
+-   ways to format your data for upload into cBioPortal.
+
+**What is cBioportal?**
+-----------------------
+**Section in progress**
+-   It is an open-source web-based platform that allows for the visualization, exploration and analysis of cancer genomics data. When you upload data from your study, the portal dynamically creates graphs that allow you to analyze your data based on clinical characteristics.
+-   The public version of this instance can be accessed here: <https://www.cbioportal.org/> and hosts publicly available/published data.
+-   It is interactive and user-friendly
+-   It allows for the easy visualization of both genomic and clinical data
+
+### What can I do with cBioportal
+
+How can it be used to facilitate genomic research and visualization.
+
+-   Links : <https://docs.cbioportal.org/user-guide/overview/>
+
+**What is the FH instance?**
+----------------------------
+
+-   It is a **local installation** of cBioPortal within the **Fred Hutch infrastructure**.
+-   Looks exactly like the [public instance](https://www.cbioportal.org/) and has all the **same features** as the **public** **instance**.
+-   It is a **secure** **place** to host your data.
+    -   cBioPortal has ben reviewed and approved by the Fred Hutch Information Security team. This permits users to include individually identifying research data such as [Personal Health Information](https://sciwiki.fredhutch.org/datascience/phi/#types-of-data-sets) (PHI) if covered in your studies [IRB](https://sciwiki.fredhutch.org/_datascience/consent_IRB/).
+-   Study **data** is only **visible** to **authorized** personnel approved by the study lead/PI.
+-   Unlike the public instance, you can upload **interim data** onto the instance. This can aid your research by not having you to generate plots and graphs that can often be time consuming.
+
+### How can I access the Fred Hutch instance of cBioportal?
+
+-   Before you are able to access the Fred Hutch instance of cBioPortal you must:
```

Comment:
Make this into a checklist using `- [ ]` in front of items

Reply:
Done. You will see this in the next draft pull.